bright-ifit-purchase-validation.yaml

```diff
@@ -181,3 +215,22 @@ paths:
           description: 'Something went wrong when attempting to entitle the user or general internal server error'
       tags:
         - Validate Purchase Transaction
+  /v1/tv-app/user-subscription-info/{userId}:
+    get:
+      summary: Returns some specific user subscription info
+      description: Provides particular user subscription info for BrightAI to cater what is displayed in the TV App purchase products catalogue
+      responses:
+        '200':
+          description: '200'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserSubscriptionInfoResponse'
+        '400':
+          description: 'User does not have an active subscription'
```

Comment:
[Slack thread](https://ifit.slack.com/archives/C01RDG6FCHL/p1622150955145900?thread_ts=1622066302.126100&cid=C01RDG6FCHL) - jeff question on this currently "returning a 400 bad request, or do you think it should just return a 200 but all the values are null 🤔 "

don't think I have a strong opinion other than what was expressed in the slack thread already. I found [this SO post](https://stackoverflow.com/questions/25239565/status-code-to-return-on-not-found-sub-resource) that mentions, "404 (Not Found) is definitely not the correct response code. The response code you want is 422 (Unprocessable Entity)." (the original poster is doing a PUT, though). But the poster right after that suggests a 404, so 🤷 .  It always feels like this rest stuff is partial guesswork, haha